src/Microsoft.AspNet.Mvc.ModelBinding/ModelMetadataTypeAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// This attribute specifies the metadata class to associate with a data model class.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
should this be inherited? This is one of the places where we usually set `Inherited = ??` because we have a hard time remembering the default.